pkgs/servers/peertube/server.nix

```diff
@@ -0,0 +1,28 @@
+{ yarnModulesConfig, mkYarnModulesFixed, sources, version, nodejs, stdenv }:
+rec {
+  modules = mkYarnModulesFixed rec {
+    inherit version;
+    pname = "peertube-server-yarn-modules";
+    name = "${pname}-${version}";
+    packageJSON = "${sources}/package.json";
+    yarnLock = "${sources}/yarn.lock";
+    pkgConfig = yarnModulesConfig;
+  };
+  dist = stdenv.mkDerivation {
+    inherit version;
+    pname = "peertube-server";
+    src = sources;
+    buildPhase = ''
+      ln -s ${modules}/node_modules .
+      patchShebangs scripts/build/server.sh
+      npm run build:server
+    '';
+
+    installPhase = ''
+      mkdir $out
+      cp -a dist $out
+    '';
+
+    buildInputs = [ nodejs ];
```

Comment:
Node is used in ``scripts/build/server.sh`` so it should be in nativeBuildInputs.

Reply:
Oh cool thanks. I pushed a change already that makes sure it's using the same nodejs version as default.nix.